src/main/webapp/help-autoCreate.html

```diff
@@ -0,0 +1,8 @@
+<div>
+	<p>
+		Creates a project in Code Dx with the specified name if one does not yet exist.
+	</p>
```

Comment:
Referred to "Code Dx" in this new doc for consistency, the Code Dx -> SRM name change is still pending throughout this plugin